eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -670,7 +671,7 @@ object Router {
   case object GetChannelsMap
   case object GetChannelUpdates
 
-  case class GetNode(nodeId: PublicKey)
+  case class GetNode(replyTo: ActorRef, nodeId: PublicKey)
```

Comment:
```suggestion
  case class GetNode(replyTo: typed.ActorRef[GetNodeResponse], nodeId: PublicKey)
```

Reply:
fixed in [bc0e1b9](https://github.com/ACINQ/eclair/pull/2575/commits/bc0e1b9a03081eaba768a9ea61fb8f3a93729ee2)